src/NHibernate/Multi/IQueryBatchItem.cs

```diff
@@ -48,7 +54,7 @@ public partial interface IQueryBatchItem
 		void ProcessResults();
 
 		/// <summary>
-		/// Immediate query execution in case dialect is non batchable
+		/// Immediate query execution in case dialect is non batched
```

Comment:
I would have not change this comment. The query execution is not batched, due to the dialect being non-batchable. Or since that is not the dialect which is batched anyway, we could also write `in case the dialect does not support batches.`. (This is not a request for change, this method is not meant to be directly used by the user anyway.)